frontend-ssr/components/Scroll/Scroll.tsx

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
We should uncomment these rules and change the code in such a way that it doesn't conflict. Don't be afraid to modify the original code - that was written in a rush and without TypeScript.

Reply:
You still left one of them 😅